src/Config/ConfigInterface.php

```diff
@@ -2,13 +2,16 @@
 
 namespace FriendsOfTwig\Twigcs\Config;
 
-use Symfony\Component\Finder\Finder;
-
 /**
  * Special thanks to https://github.com/c33s/twigcs/ which this feature was inspired from.
+ *
+ * @method string getDisplay()
```

Comment:
If this interface realy implement this method, add the function in the interface.

Reply:
I added it there to keep BC, is there any BC promise? or only in the use of the command?

Reply:
Yes we want to avoid BC breaks on the config. We'll add this to the interface on the next major release (not planned yet though).